src/Compilers/CSharp/Portable/CodeGen/EmitExpression.cs

```diff
@@ -1826,7 +1826,7 @@ private void EmitArrayCreationExpression(BoundArrayCreation expression, bool use
             EmitPopIfUnused(used);
         }
 
-        private void EmitStackAllocArrayCreationExpression(BoundStackAllocArrayCreation expression, bool used)
+        private void EmitConvertedStackAllocExpression(BoundConvertedStackAllocExpression expression, bool used)
         {
             EmitExpression(expression.Count, used: true);
```

Comment:
I know this is existing code, but it looks wrong, and if so this is a serious bug. According to the specification for the `localloc` instruction, its operand is the number of *bytes* to be allocated. However, in the bound node the `Count` field is the number of elements of the array to be allocated. Unless it is an array of `byte`, these are not the same thing.

Reply:
I see that the count is rewritten in lowering.